### PR TITLE
updates to allow json to include handlebars style expressions

### PR DESCRIPTION
**Pull request by moloko**
This allows for course authors to use these in their JSON to display data from the models that are available to the template, specifically it's to allow for the functionality described in #1040 whereby users can pull in data from SCORM by using expressions like `{{_globals._learnerInfo.firstname}}` in their JSON, giving you the option to do things like this:

2017-03-27 18_14_59-adapt version 2 0 demonstration _ presentation components

It also introduces a new way of getting the `_globals` object into handlebars templates - via an `{{import_globals}}` expression at the top of each template - meaning we can get rid of [this line in adaptView.js](https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/views/adaptView.js#L16)... only question is whether we do that immediately or leave it in place for a bit to cover people who use the latest version of the framework with a old version of a theme..?

Note that I will also be updating adapt-contrib-spoor to get it to replace the default values in `_globals._learnerInfo` with the actual data from the LMS.

I will also need to amend the copies of the handlebars templates that are stored in adapt-contrib-vanilla.

### CODE REVIEW

File: src/core/js/views/adaptView.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ define([
         initialize: function() {
             this.listenTo(Adapt, 'remove', this.remove);
             this.listenTo(this.model, 'change:_isVisible', this.toggleVisibility);
-            this.model.set('_globals', Adapt.course.get('_globals'));
```
Comment (oliverfoster) [inline src/core/js/views/adaptView.js]: removing this line is going to break all existing themes

Comment (moloko) [inline src/core/js/views/adaptView.js]: hey, it was your suggestion ;-)

Comment (oliverfoster) [inline src/core/js/views/adaptView.js]: :P if we leave it in place for the moment, it can come out in v3

**Comment by oliverfoster** [comment]
The ``{{import_globals}}`` can be used in any handlebars, not just those which belong to a view that extend ``AdaptView``, which is a nice change. It doesn't matter so much that the ``_globals`` object will be included in the ``handlebars`` data context twice - once in ``AdaptView`` and once from the global helper. Not breaking the themes I would say is more important for the moment.

**Comment by oliverfoster** [comment]
I would like to remove the templates from the theme.

**Pull request merged**